.github/dependabot.yml

```diff
@@ -13,3 +13,10 @@ updates:
       interval: "daily"
       time: "06:27"
       timezone: "Europe/Warsaw"
+
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
```

Comment:
In fact, due to a recent [change](https://github.blog/changelog/2021-06-16-dependabot-now-schedules-version-updates-uniformly/) in how Dependabot schedules updates, I believe it's best to delete the `schedule` section altogether. What do you think?